lib/ncdata/xarray.py

```diff
@@ -96,6 +107,16 @@ def store(
 
         # Install variables, creating dimensions as we go.
         for varname, var in new_variables.items():
+            if isinstance(var.data, np.ndarray):
+                # Zarr2 metadata
+                if "axis" not in var.attrs:
+                    std_axes = ["latitude", "longitude", "time"]
+                    if not list(set(var.attrs.values()) & set(std_axes)):
```

Comment:
```suggestion
                    if not [s for s in std_axes if s in str(var.attrs.values())]:
```

@valeriupredoi suggestion pulled in from https://github.com/pp-mo/ncdata/pull/147#discussion_r2266986476

Reply:
Also copied here from [your comment](https://github.com/pp-mo/ncdata/pull/147#discussion_r2266996331) ...

> this will fix that failing test, and all the almost 1000 tests pass now 😃 The problem is that this method (and the previous approach) are not 100% exhaustive in figuring out which are coords that should have realized data, so a few of these warnings will be raised for those that the check misses them, but beats me how I can create a bulletproof check, given the variety of names and metadata, and differences between Zarr2 and Zarr3 (alas, most of them don't raise the warning)

Reply:
gotcha, cheers!